### PR TITLE
Standardize Icon Components and Add to Storybook

### PR DESCRIPTION
**Pull request by besampson0**
Resolves [Caseflow - 2548](https://vajira.max.gov/browse/CASEFLOW-2548): Standardize icon components and add to Storybook.

### Description
Standardize the icon components so that they each accept size, color, and cname props. Some icons take more than this, but they should all at least take size and color. They should all be added to storybook as well so that it is clear how icons are being used throughout the project. 

### Acceptance Criteria
- [ ] Code compiles correctly
- [ ] Each Icon is its own component
- [ ] Each Icon has been added to storybook

### Testing Plan
1. Run 'yarn storybook'
2. View each icon

### CODE REVIEW

File: client/app/components/icons/LinkSymbol.jsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const LinkSymbol = (props) => {
```
Comment (jcq) [inline client/app/components/icons/LinkSymbol.jsx:4]: I don't seem to see a story for `LinkSymbol`...?

File: client/app/components/icons/MissingSymbol.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// Not used anywhere
```
Comment (jcq) [inline client/app/components/icons/MissingSymbol.jsx:4]: Good note, though also something that might not get updated if/when it ever is used... 🤔 

Anyhoo, still good to note 😅

File: client/app/components/icons/Trashcan.jsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  color: PropTypes.string,
+
+  /**
+  * Adds class to the component. Default value is ''.
+  */
+  cname: PropTypes.string
+};
+Trashcan.defaultProps = {
+  size: 26,
+  color: '',
```
Comment (jcq) [inline client/app/components/icons/Trashcan.jsx:58]: It looks like this has a `stroke` color, which is blue. Does it make sense to separate those from fill color somehow...? What if someone wants a trash can that isn't a blue outline?

**Review by jcq — COMMENTED** [review]
Great work — huge improvements in documentation, etc

File: client/app/2.0/components/reader/DocumentList/DocumentsTable/Columns.jsx [hunk 4]
```diff
@@ -9,8 +9,8 @@ import Button from 'app/components/Button';
 import DropdownFilter from 'app/components/DropdownFilter';
 import ViewableItemLink from 'app/components/ViewableItemLink';
 import { Highlight } from 'components/reader/DocumentList/Highlight';
-import FilterIcon from 'app/components/FilterIcon';
-import { DoubleArrow } from 'app/components/RenderFunctions';
+import FilterIconOutline from 'app/components/FilterIconOutline';
```
[the remark this answers is not in the document]
Comment (besampson0) [reply client/app/2.0/components/reader/DocumentList/DocumentsTable/Columns.jsx:12]: yes, yes it should :)

File: client/app/2.0/components/reader/DocumentViewer/Header.jsx [hunk 5]
```diff
@@ -6,14 +6,12 @@ import PropTypes from 'prop-types';
...
-  Rotate,
-  SearchIcon,
-  DownloadIcon
-} from 'app/components/RenderFunctions';
+import { LeftChevron } from 'app/components/icons/LeftChevron';
+import { ExternalLink } from 'app/components/icons/ExternalLink';
+import { FitToScreen } from 'app/components/icons/FitToScreen';
+import { Rotate } from 'app/components/icons/Rotate';
+import { SearchIcon } from 'app/components/icons/SearchIcon';
+import { DownloadIcon } from 'app/components/icons/DownloadIcon';
```
[the remark this answers is not in the document]
Comment (besampson0) [reply client/app/2.0/components/reader/DocumentViewer/Header.jsx:14]: Agreed.

File: client/app/components/icons/CategoryIcon.jsx [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+  /**
+   * Sets the color of the component. Default color is white.
+   */
+  color: PropTypes.string,
+
+  /**
+   * Sets height of the component, width is set automatically by the svg viewbox property. Default height is '20px'.
+   */
+  size: PropTypes.number,
```
[the remark this answers is not in the document]
Comment (besampson0) [reply client/app/components/icons/CategoryIcon.jsx:32]: I really like this suggestion.

File: client/app/components/icons/ClockIcon.jsx [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+  color: PropTypes.string,
+
+  /**
+   * Adds class to the component. Default value is 'svg-inline--fa fa-clock fa-w-16'.
+   */
+  cname: PropTypes.string
+};
+ClockIcon.defaultProps = {
+  size: 16,
+  color: 'currentColor',
```
[the remark this answers is not in the document]
Comment (besampson0) [reply client/app/components/icons/ClockIcon.jsx:33]: I wasn't really sure at first, doesn't look like it does anything. Changing it to be like the others.

File: client/app/components/icons/CategoryIcon.jsx (hunk 6 above)
[the remark this answers is not in the document]
Comment (besampson0) [reply client/app/components/icons/CategoryIcon.jsx:32]: There are variables for colors we should be using throughout. Will add this to the list.

[the remark this answers is not in the document]
Comment (besampson0) [reply client/app/components/icons/CategoryIcon.jsx]: Will be addressed in [CF-2583](https://vajira.max.gov/browse/CASEFLOW-2583)

File: client/app/components/icons/ArrowDown.jsx [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+  /**
+     * sets color of the component. Default value is 'COLORS.GREY_MEDIUM'.
+     */
+  color: PropTypes.string,
+
+  /**
+    * Adds class to the component. Default value is ''.
+    */
+  cname: PropTypes.string
```
[the remark this answers is not in the document]
Comment (besampson0) [reply client/app/components/icons/ArrowDown.jsx:42]: Will be resolved in [CF-2583](https://vajira.max.gov/browse/CASEFLOW-2583)

File: client/app/2.0/components/reader/DocumentViewer/Header.jsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (besampson0) [reply client/app/2.0/components/reader/DocumentViewer/Header.jsx:14]: Will be resolved in [CF-2583](https://vajira.max.gov/browse/CASEFLOW-2583)

**Pull request closed without merging**